server.js

```diff
@@ -4,7 +4,7 @@ var sneakernet = require('hyperlog-sneakernet-replicator')
 var net = require('net')
 
 var body = require('body/any')
-var qs = require('querystring')
+var parseUrl = require('url').parse
```

Comment:
This was the only use site of `querystring`; you can drop it from `package.json`.